src/main/kotlin/com/exactpro/th2/netty/bytebuf/util/ByteBufDataUtil.kt

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@file:JvmName("ByteBufDataUtil")
+
+package com.exactpro.th2.netty.bytebuf.util
+
+import io.netty.buffer.ByteBuf
+import java.nio.charset.Charset
+import java.util.*
+
+const val DEFAULT_END_CHAR = '\u0000'
```

Comment:
Maybe rename to `DEFAULT_PAD_CHAR`?
I would also change it to space instead of zero